src/generate.ts

```diff
@@ -36,16 +57,16 @@ function getColorTheme(color: ColorOptions) {
 
   if (typeof color === 'string') {
     name = color
-    ;({ light, dark } = getBuiltInTheme(color))
+    ; ({ light, dark } = getBuiltInTheme(color))
```

Comment:
Can you help me remove these formatting changes?

Reply:
Okay, that wasn't expected haha, might be a bit sleepy yesterday